pyrokinetics/gk_code/GKInputGS2.py

```diff
@@ -212,6 +212,99 @@ def get_local_species(self):
         local_species.normalise()
         return local_species
 
+    def _read_single_grid(self):
+        return {
+            "nky": 1,
+            "nkx": 1,
+            "ky": self.data["kt_grids_single_parameters"]["aky"] / sqrt2,
+            "kx": 0.0,
```

Comment:
Should this be `kx = ky*shat*theta0`?

Reply:
(Note that this is unchanged from existing implementation)

Maybe:

```suggestion
            "kx": self.data["kt_grids_single_parameters"].get("akx", ky * shat * theta0) / sqrt2
```

We also don't handle being able to set `n0` and `rhostar_single` here, but it looks like we probably actually _can't_?

Reply:
We store `theta0` in numerics but not in the output grid. This probably should be what is suggested